files/src/main/scalajvm/sttp/tapir/files/model.scala

```diff
@@ -0,0 +1,78 @@
+package sttp.tapir.files
+
+import sttp.model.MediaType
+import sttp.model.headers.{ETag, Range}
+
+import java.time.Instant
+
+case class StaticInput(
+    path: List[String],
+    ifNoneMatch: Option[List[ETag]],
+    ifModifiedSince: Option[Instant],
+    range: Option[Range],
+    acceptEncoding: Option[String]
+) {
+
+  def acceptGzip: Boolean = acceptEncoding.contains("gzip")
+}
+
+trait StaticErrorOutput
+object StaticErrorOutput {
+  case object NotFound extends StaticErrorOutput
+  case object BadRequest extends StaticErrorOutput
+  case object RangeNotSatisfiable extends StaticErrorOutput
+}
+
+trait HeadOutput
```

Comment:
maybe we can simply have `type HeadOuput.Found  = StaticOuput.Found[Unit]`? the head enpoint could then have `emptyOutputAs[Unit]` for the body placeholder

Reply:
Sounds doable, I'll work on it.

Reply:
It worked! Although I can't create an alias with a dot like this, I removed HeadOutput entirely, and long the definition of `staticHeadEndpoint` is now replaced with a simple
```scala
    staticEndpoint(endpoint.head, emptyOutput)
```
Thanks for the tip!